### PR TITLE
Fix not printing check transaction stderr

### PR DESCRIPTION
**Pull request by Rezney**

### CODE REVIEW

File: repos/system_upgrade/el7toel8/files/rhel_upgrade.py [hunk 1]
```diff
@@ -90,7 +90,10 @@ def run(self):
         self.base.distro_sync()
 
         if self.opts.tid[0] == 'check':
-            self.base.resolve(allow_erasing=self.cli.demands.allow_erasing)
+            try:
+                self.base.resolve(allow_erasing=self.cli.demands.allow_erasing)
+            except Exception as e:
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/files/rhel_upgrade.py]: Could you use more specific exception(s)?

Comment (Rezney) [reply repos/system_upgrade/el7toel8/files/rhel_upgrade.py]: ouch, I was almost sure I was opening this PR just against my own fork to show @vinzenz the diff. But anyway I will have to test it, so setting WIP...

File: repos/system_upgrade/el7toel8/files/rhel_upgrade.py [hunk 2]
```diff
@@ -90,7 +91,11 @@ def run(self):
         self.base.distro_sync()
 
         if self.opts.tid[0] == 'check':
-            self.base.resolve(allow_erasing=self.cli.demands.allow_erasing)
+            try:
+                self.base.resolve(allow_erasing=self.cli.demands.allow_erasing)
+            except dnf.exceptions.DepsolveError as e:
+                print(e, file=sys.stderr)
+                raise dnf.exceptions.DepsolveError
```
Comment (vinzenz) [inline repos/system_upgrade/el7toel8/files/rhel_upgrade.py]:
```suggestion
                raise
```

File: repos/system_upgrade/el7toel8/files/rhel_upgrade.py (hunk 1 above)
Comment (Rezney) [reply repos/system_upgrade/el7toel8/files/rhel_upgrade.py]: fixed

File: repos/system_upgrade/el7toel8/files/rhel_upgrade.py [hunk 3]
```diff
@@ -90,7 +91,11 @@ def run(self):
         self.base.distro_sync()
 
         if self.opts.tid[0] == 'check':
-            self.base.resolve(allow_erasing=self.cli.demands.allow_erasing)
+            try:
+                self.base.resolve(allow_erasing=self.cli.demands.allow_erasing)
+            except dnf.exceptions.DepsolveError as e:
+                print(e, file=sys.stderr)
```
Comment (vinzenz) [inline repos/system_upgrade/el7toel8/files/rhel_upgrade.py]:
```suggestion
                print(str(e), file=sys.stderr)
```

**Pull request merged**